web/concrete/src/Application/Application.php

```diff
@@ -546,4 +585,16 @@ public function build($concrete, $parameters = array())
         return $object;
     }
 
+    public function setupClearCacheEvents()
+    {
+        \Events::addListener('on_page_delete', array($this, 'partialClearCaches'));
```

Comment:
Same here, this shouldn't be using the facade or the class alias. `$events = $this['director']; $events->addListener(...)`